hondana/client.py

```diff
@@ -1935,6 +1935,7 @@ async def user_list(
             A returned collection of users.
         """
         innert_limit = limit or 10
+        # Not sure if this should be `inner_limit` or something.
```

Comment:
Let me know about this part, leaving this comment incase you accidentally miss it.

Reply:
This should be `inner_limit` yeah. Thank you.

Reply:
Alright ill get that changed in the next commit.